### PR TITLE
Few security fixes

### PR DESCRIPTION
**Pull request by mahajanleena1995**
- Nonce verification in all the Reset options
- Check ABSPATH in index.php

### CODE REVIEW

File: admin/index.php [hunk 1]
```diff
@@ -112,7 +112,7 @@ function rich_snippet_dashboard() {
 									<div class="table">
 										<p>' . wp_kses_post( __( 'Strings to be displayed on frontend for <strong>Item Review Rich Snippets &mdash;</strong>', 'all-in-one-schemaorg-rich-snippets' ) ) . '</p>
 										<form id="bsf_review_form" method="post">
-										' . wp_nonce_field( 'snippet_review_form_action', 'snippet_review_nonce_field' ) //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped 
+										' . wp_nonce_field( 'snippet_review_form_action', 'snippet_review_nonce_field' ) //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```
Comment (patilvikasj) [inline admin/index.php:115]: Remove PHPCS ignore and use escaping.

File: admin/index.php [hunk 2]
```diff
@@ -154,8 +154,8 @@ function rich_snippet_dashboard() {
 									<div class="table">
 										<p>' . wp_kses_post( __( 'Strings to be displayed on frontend for <strong>Events Rich Snippets &mdash;</strong>', 'all-in-one-schemaorg-rich-snippets' ) ) . '</p>
 										<form id="bsf_event_form" method="post">
-										' . wp_nonce_field( 'snippet_event_form_action', 'snippet_event_nonce_field' ) //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped 
-										. ' 
+										' . wp_nonce_field( 'snippet_event_form_action', 'snippet_event_nonce_field' ) //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```
Comment (patilvikasj) [inline admin/index.php:157]: Same here. Remove PHPCS ignore comment.

**Review by patilvikasj — CHANGES_REQUESTED** [review]
Please check comments @mahajanleena1995

File: admin/index.php (hunk 1 above)
Comment (mahajanleena1995) [reply admin/index.php]: Done

File: admin/index.php (hunk 2 above)
Comment (mahajanleena1995) [reply admin/index.php]: Done

**Review by patilvikasj — APPROVED** [review]

**Pull request merged**